pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -165,15 +175,24 @@ func (d *Deployment) ReportSinceLastUpdated() string {
 	for _, p := range d.pods {
 		if s := p.ActionableError().Message; s != "" {
 			result.WriteString(fmt.Sprintf("%s %s %s: %s\n", tab, tabHeader, p, s))
-			for _, l := range p.Logs() {
-				result.WriteString(fmt.Sprintf("%s\n", l))
+			// if, is muted then write container logs to file and last 3 lines to
+			// result.
+			out, writeTrimLines, err := withLogFile(p.Name(), &result, p.Logs(), isMuted)
+			if err != nil {
+				logrus.Debugf("could not create log file %v", err)
+			}
+			trimLines := []string{}
+			for i, l := range p.Logs() {
+				formattedLine := fmt.Sprintf("%s %s > %s\n", tab, tab, strings.TrimSuffix(l, "\n"))
+				if isMuted && i >= len(p.Logs())-3 {
```

Comment:
i don't love that `3` is hardcoded everywhere, especially across multiple files. can you pull it out as a constant? we might also want to expose it as configurable at some point.

Reply:
sorry, i don't know how i missed moving this to a constant. I had it here initially for mocking output.